src/components/Card.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  width: 340px;
+  border-radius: 15px;
+  box-shadow: 0px 5px 25px 0px rgba(0, 0, 0, 0.1);
+  margin: 0;
+`;
+
+const Link = styled.a`
+  text-decoration: none;
+`;
+
+const Img = styled.img`
+  width: 340px;
+  height: 200px;
+  border-radius: 15px 15px 0px 0px;
+  margin: 0;
+`;
+
+const Desc = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 10px;
+  width: 300px;
+  height: 100px;
+  border-radius: 0px 0px 15px 15px;
+  margin: 0px;
+  padding: 15px 20px;
+`;
+
+const Ago = styled.span`
+  color: #666666;
+  font-size: 0.8rem;
+  margin: 0px;
+`;
+
+const Content = styled.p`
+  font-size: 1rem;
+  margin: 0px;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const Date = styled.span`
+  color: #333333;
+  font-size: 0.85rem;
+  margin: 0;
+`;
+
+function Card({ url, src, desc, createdAt }) {
+  const processedCreatedAt = createdAt.substr(0, 10);
+
+  return (
+    <>
+      <Container>
+        <Link href={url}>
+          <Img src={src} alt='대표 이미지' />
+          <Desc>
+            <Ago>dateNow - 'createdAt'</Ago>
+            <Content>
+              {desc.length > 10 ? desc.substr(0, 70) + '...' : desc}
```

Comment:
로직을 return 부에 담게되면 계속 연산을 하게 되어서 비추천드립니다!
useMemo 로 위로 빼서 변수로 가져오시는 걸 추천드려요!
```suggestion
              {desc.length > 10 ? subDesc : desc}
```

Reply:
조건부 렌더링을 하려고 했던 의도였는데,
로직 자체를 return 안에 담게 되면, 계속 연산을 하게 되는군요!! 
수정했습니다 감사합니다!! 👍